hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -148,8 +141,8 @@ private void startNio() {
 
             NioAsyncServerSocket serverSocket = NioAsyncServerSocket.open(eventloop);
             serverSockets.add(serverSocket);
-            int receiveBufferSize = clientSocketConfig.getReceiveBufferSize();
-            int sendBufferSize = clientSocketConfig.getSendBufferSize();
+            int receiveBufferSize = clientSocketConfig.getReceiveBufferSizeKb();
```

Comment:
Should this not be multiplied by 1024? The serverSocket.receiveBufferSize expects in bytes and getReceiveBufferSizeKb (should be KB otherwise it is kilobit) returns in kilobyte.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/5f4a69eb4a06c2244821276e1d78745968efa3a9

Reply:
I used getReceiveBufferSizeKb instead of getReceiveBufferSizeKB because thats how we do it in the EndpointConfig and I tried to be consistent with that.

Reply:
Let's use the correct name.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/c927ebcc90f6823ddee9ae30fafa66e5471087d7